src/intrinsics/node/buffer.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow */
+
+import invariant from "../../invariant.js";
+import { Realm } from "../../realm.js";
+import { Value, NumberValue, NativeFunctionValue, ObjectValue, StringValue } from "../../values/index.js";
+import { Set, ToInteger } from "../../methods/index.js";
+
+declare var process: any;
+
+function getBufferFromArg(realm: Realm, value: Value): Uint8Array {
```

Comment:
Yes, that is actually preferable because it makes it easier for the reader to verify that the invariant is true.